lib/puppet-strings/version.rb

```diff
@@ -0,0 +1,3 @@
+module PuppetStrings
+  VERSION = '2.1.0'.freeze
```

Comment:
does this need to be 'v2.1.0' to match the new tags?

Reply:
The version is `'2.1.0'`, the `'v'` is added in the Rake task.